_data/simple-icons.json

```diff
@@ -15100,6 +15100,16 @@
             "hex": "FFA633",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Sway",
+            "hex": "68751C",
```

Comment:
@simple-icons/pr-reviewers What do you guys think about the choice of color for this multi-color logo?